Makefile

```diff
@@ -10,6 +10,11 @@ GO_BUILD_ARGS=CGO_ENABLED=0 GO111MODULE=on
 # TODO (suhanime): Pin go versions and lint
 # TODO (suhanime): Enable linting when we don't have unimplemented methods
 
+# The golang 1.13 image used in CI enforces vendoring. Workaround that by unsetting it.
+ifeq ($(GOFLAGS), -mod=vendor)
```

Comment:
Can goflags include other flags, not just the vendor flag? If so, I don't think that will  be caught here. In that case you will need to keep the other flags intact, removing only the vendor flag

Reply:
@ravisantoshgudimetla and I had the same discussion with @mansikulkarni96 today. The ART team recommends doing it this way in CI. PS: we have done this in a few places now without any issues.

Reply:
Mentioning the discussion commit [here](https://github.com/openshift/release/pull/6337/commits/db916acf9afac0b1dd883a159a80696132f244e7) for future reference.